src/components/listings/ListingCardGrid.tsx

```diff
@@ -21,31 +22,45 @@ function ListingCardGrid(props: ListingCardGridProps) {
 		>
 			<div
 				style={{
+					marginBottom: 16,
+					display: "flex",
+					gap: 16,
+				}}
+			>
+				<span style={{ fontSize: 24, whiteSpace: "nowrap" }}>
+					Found {props.dataList.length} listings in Richardson, TX
+				</span>
+				<Form.Control placeholder="Search other locations..." />
+			</div>
+			<div
+				style={{
+					width: "100%",
 					marginBottom: 32,
 					display: "flex",
 					flexWrap: "wrap",
 					gap: 16,
 				}}
 			>
+				<FilterField label="Avaliable Start" type="date" />
+				<FilterField label="Avaliable Until" type="date" />
 				<FilterField label="Min. Rent" />
 				<FilterField label="Max. Rent" />
 				<FilterSelect
-					label="Type of Place"
+					label="Pet Friendly"
 					options={[
-						{ title: "Apartment", value: "apartment" },
-						{ title: "House", value: "house" },
-						{ title: "Condo", value: "condo" },
+						{ title: "Yes", value: true },
```

Comment:
Yes, if most of the options are using the same options, it would be great to extract and reuse that object. Again this is a very optional change. If you prefer it this way, you can choose to ignore :)